src/Application/Game.cpp

```diff
@@ -147,7 +147,12 @@ std::string FindMm7Directory() {
 int Game::Run() {
     IntegrityTest();
 
-    SetDataPath(FindMm7Directory());
+    std::string mm7dir = FindMm7Directory();
+    if(mm7dir.empty()) {
+        log->Warning("MM7 directory not found, please set WOMM_PATH_OVERRIDE environment variable");
+        return -1;
```

Comment:
That breaks perfectly valid case when executable was launched from game directory...
This should be made non-critical error and reworded to something like 'MM7 directory not found, consider using WOMM_PATH_OVERRIDE environment variable, launching with expectation that current directory is MM7 directory..."

Reply:
Thanks, fixed